### PR TITLE
WIP: Add shellcheck to cicd

### PR DESCRIPTION
**Pull request by nikolaishields**

### CODE REVIEW

**Comment by nikolaishields** [comment]
Failures seen in drone match those that occurred during local testing. Fixes to individual shell scripts will follow in a subsequent pr.

**Review by briandowns — APPROVED** [review]

File: bin/update_go [hunk 1]
```diff
@@ -38,7 +38,7 @@ get_version() {
     for i in "Dockerfile" "Dockerfile.amd64"; do
         if [ -e  "${repo_path}/${i}" ]; then
              info ${repo_path}
```
Comment (crosas-3p) [inline bin/update_go:40]: nit: shellcheck suggests to double quote variables info "${repo_path}"

**Review by crosas-3p — APPROVED** [review]
lgtm, a few small things that can be homologated but overall is ok

File: bin/update_go (hunk 1 above)
Comment (nikolaishields) [reply bin/update_go:40]: Presently the CI is exclusively checking for warnings, and this flagged error is classified under the log level info.
I'll add support for checking the other log levels and address the necessary changes on line 40 in  another pr shortly.
Thanks @crosas=3p!

Comment (nikolaishields) [reply bin/update_go:40]: See #42 for resolution.

**Pull request merged**